doc/rados/configuration/osd-config-ref.rst

```diff
@@ -505,6 +505,22 @@ recovery operations to ensure optimal performance during recovery.
 :Type: 32-bit Integer
 :Default: ``5``
 
+
+``osd agent max ops``
+
+:Description: The maximum number of agent flush ops for tiering flushing.
+:Type: 32-bit Integer
+:Default: ``4``
+
+
+``osd agent max low ops``
+
+:Description: The maximum number of low speed agent flush ops for tiering flushing.
```

Comment:
also, could you add a hyper link to https://github.com/ceph/ceph/blob/master/doc/rados/operations/pools.rst#user-content-cache-target-dirty-high-ratio ?